source/nodejs/adaptivecards/src/utils.ts

```diff
@@ -272,3 +272,21 @@ export function addCancelSelectActionEventHandler(element: HTMLElement) {
         e.cancelBubble = true;
     };
 }
+
+export function SetDateOrTimePickerVisibility(input: HTMLInputElement, value: string | undefined, show: boolean) {
```

Comment:
Very confused about what this logic achieves and why it's necessary. Could you explain?